doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -5,23 +5,25 @@
 	</brief_description>
 	<description>
 		[EditorTranslationParserPlugin] is invoked when a file is being parsed to extract strings that require translation. To define the parsing and string extraction logic, override the [method _parse_file] method in script.
-		Add the extracted strings to argument [code]msgids[/code] or [code]msgids_context_plural[/code] if context or plural is used.
-		When adding to [code]msgids_context_plural[/code], you must add the data using the format [code]["A", "B", "C"][/code], where [code]A[/code] represents the extracted string, [code]B[/code] represents the context, and [code]C[/code] represents the plural version of the extracted string. If you want to add only context but not plural, put [code]""[/code] for the plural slot. The idea is the same if you only want to add plural but not context. See the code below for concrete examples.
+		The return value should be an [Array] of [PackedStringArray]s, one for each translated string. Each translation entry should contain [code][msgid, msgctxt, msgid_plural, context][/code], where all except [code]msgid[/code] are optional. Empty strings will be ignored.
 		The extracted strings will be written into a POT file selected by user under "POT Generation" in "Localization" tab in "Project Settings" menu.
 		Below shows an example of a custom parser that extracts strings from a CSV file to write into a POT.
 		[codeblocks]
 		[gdscript]
 		@tool
 		extends EditorTranslationParserPlugin
 
-		func _parse_file(path, msgids, msgids_context_plural):
+		func _parse_file(path):
+		    var ret = []
```

Comment:
```suggestion
		    var ret: Array[PackedStringArray] = []
```

Even though we don't use static typing in the documentation, I think the type hint is necessary here because a typed array cannot reference an untyped one.